### ISSUE TITLE
Xrootd Cache Summary Data has a typo in `rd.hits`

### ISSUE THREAD

**Issue by haoming29** [xrootd/xrootd#2219]
In the summary packet, if cache is enabled, the `cache` data has a sub-attribute `rd.hits`, which should be an `int64` value according to documentation [here](https://xrootd.slac.stanford.edu/doc/dev55/xrd_monitoring.htm#_Toc99653733).

However, a printed summary packet gives: 

```xml
<rd>
	<in>0</in>
	<out>0</out>
	<hits>0></hits>
	<miss>0</miss>
</rd>
```

where there is an extra `>` after the value in `<hits>`. This causes error when deserializing the packet back to XML.

The same typo appears on the documentation as well, where it records:

```xml
<hits>int64></hits>
```

**Comment by abh3** [comment]
Thank you for the report! This will be corrected in 5.7.0, our next release.

**Comment by abh3** [comment]
Fix is now in devel to be included in the next release.
